src/components/BurgerMenu/StoreMenuOptions.js

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import { List, ListItemText, ListItem } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { auth } from "../../firebase";
+import {
+  ADMIN_QUEUE_MANAGEMENT_PATH,
+  QUEUE_POSTER_PATH,
+  ADMIN_END_QUEUE_PATH,
+  ADMIN_HOME_ONBOARDING_PATH,
+  ABOUT_US_PATH,
+  ADMIN_LOGIN_PATH
+} from "../../constants/RoutesConstants";
+import { useLocation, useHistory } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const useStyles = makeStyles({
+  list: { paddingTop: "30px" },
+
+  listItem: {
+    fontSize: "24px",
+    lineHeight: "30px",
+    color: "#ffffff"
+  },
+  listItemActive: {
+    fontSize: "24px",
+    lineHeight: "30px",
+    color: "#ffffff",
+    fontWeight: 900
+  },
+  divider: {
+    borderBottom: "2px solid white"
+  }
+});
+const StoreMenuOptions = ({ toggleMenu, queueId, user }) => {
+  const classes = useStyles();
+  const history = useHistory();
+  const { pathname } = useLocation();
+  const [t] = useTranslation();
+
+  const handleOptionClick = link => {
+    history.push(link);
+    toggleMenu();
+  };
+
+  const logout = () => {
+    toggleMenu();
+    auth.signOut().then(() => {
+      window.location.href = "/admin";
+    });
+  };
+  return (
+    <List
+      component="nav"
+      aria-label="store poster queue"
+      classes={{ root: classes.list }}
+    >
+      <ListItem
+        divider={!user}
+        classes={{ divider: classes.divider }}
+        disableGutters
+        button
+        onClick={() => handleOptionClick(ADMIN_QUEUE_MANAGEMENT_PATH)}
+      >
+        <ListItemText
+          classes={{
+            primary:
+              pathname === ADMIN_QUEUE_MANAGEMENT_PATH
+                ? classes.listItemActive
+                : classes.listItem
+          }}
+          primary={t("burger#your_store")}
+        />
+      </ListItem>
+      {queueId != null && (
+        <>
+          <ListItem
+            disableGutters
+            button
+            onClick={() =>
+              handleOptionClick(
+                `${QUEUE_POSTER_PATH.replace(":queueId", queueId)}`
+              )
+            }
+          >
+            <ListItemText
+              classes={{
+                primary:
+                  pathname === `${QUEUE_POSTER_PATH}/${queueId}`
```

Comment:
Are you sure this is doing what you want? Because `QUEUE_POSTER_PATH` has this dynamic value `:queueId`.

Reply:
Yeah, you need to replace `:queueId` in `QUEUE_POSTER_PATH` to get that intended behaviour

Reply:
I believe that the poster page should not even have the burguer menu, right?

Reply:
fixed.